modules/tide_site_alert/tide_site_alert.module

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @file
+ * Contains tide_site_alert.module.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_theme().
+ */
+function tide_site_alert_theme() {
+  return [
+    'site_alert' => [
+      'template' => 'tide-site-alert',
+    ],
+  ];
+}
+
+/**
+ * Implements hook_form_alter().
+ */
+function tide_site_alert_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if ($form_id === 'site_alert_add_form' || $form_id === 'site_alert_edit_form') {
```

Comment:
Usually better to refactory your if statements so you can return immediately and then have the code nested lower. It's a bit of an anti-pattern to have your entire function nested in an if statement

Reply:
This reminds me that I should switch to using another hook!